frontend/src/components/Warning/Warning.tsx

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react'
+import { useEvent } from '/src/hooks'
+import { Modal, Button } from '/src/components'
+
+export const Warning = () => {
+  const [isOpen, setIsOpen] = useState(false)
+  const [message, setMessage] = useState('')
```

Comment:
nit: Perhaps message does not need to use state? I think it would be fine to pass it as a prop as it won't change while the modal is rendered

Reply:
Nevermind I was misunderstanding how this works let me take a closer look at this